plugin/Gui/AdminPageFormSettingsContact.php

```diff
@@ -120,9 +120,9 @@ protected function buildForms()
 			if (empty($pInputModelBuilder->getValues())) {
 				$pInputModel->setValue(true);
 			}
-			$linkLabel = esc_html__("Request Manager", 'onoffice-for-wp-websites');
-			$linkTag = "<a href='https://de.enterprisehilfe.onoffice.com/category/additional-modules/request-manager/?lang=en'>%s</a>";
-			$link = sprintf($linkTag, $linkLabel);
+			$linkLabel = esc_html__('Request Manager', 'onoffice-for-wp-websites');
+			$linkUrl = esc_html__('https://de.enterprisehilfe.onoffice.com/category/additional-modules/request-manager/?lang=en', 'onoffice-for-wp-websites');
+			$link = sprintf("<a href='%s' target='_blank'>%s</a>", $linkUrl, $linkLabel);
```

Comment:
There is a small security risk with this, so it's best to add a [`rel="noopener"`](https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types/noopener).

Since only [89.63 % of users](https://caniuse.com/mdn-html_elements_a_implicit_noopener) have browsers that implicitely add this, I think it is better to explictely add this for now.

Reply:
yes, i added rel in tag a, can you check PR help me !

Reply:
If you want me to review/test, please request a review. ;)

Reply:
sorry, i requested you review help me !

Reply:
No worries, thanks! I will look at it soon. 👍